Task1.1/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nodejsglobalmentoringprogram",
+  "version": "1.0.0",
+  "description": "Node Js Mentoring",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Add script for babel for compiling your ES6 code.

Reply:
Not required